cpython/download_pypi_top.py

```diff
@@ -22,21 +22,24 @@
 import traceback
 import json
 import os
-import sys
 import time
 
+from termcolor import cprint
```

Comment:
Would it be possible to fall back to print() if termcolor is not installed?

```
try:
    ...
except ImportError:
    print("termcolor is missing, install it with: python -m pip install --user termcolor", file=sys.stderr)
    def cprintf(msg, *ignored):
        print(msg)
```

Reply:
Good idea, updated!